.devcontainer.json

```diff
@@ -21,10 +21,10 @@
         "iterative/example-get-started-experiments": {
           "permissions": "write-all"
         },
-        "iterative/example-get-started-frozen": {
+        "iterative/example-get-started-fixture": {
           "permissions": "write-all"
         },
-        "iterative/example-get-started-experiments-frozen": {
+        "iterative/example-get-started-experiments-fixture": {
```

Comment:
do we need it to be able to generate live experiments? I'm a bit concerned since I think we need to start using versioning for the fixture (to avoid breaking tests)

Reply:
I think it is actually ok to drop it. As long as I have GPU to generate the "demo" one, I can do the push to this (or versioned) fixture by fetching the new demo.